src/tools/schema2cpp.ts

```diff
@@ -177,6 +180,9 @@ class CppGenerator implements ClassGenerator {
     this.stringify.push(`if (entity.${valid}) printer.add("${field}: ", entity.${field}_);`);
   }
 
+  generatePredicates() {
```

Comment:
Maybe add a TODO w/ a Github issue to track this cpp generation for the future?

Reply:
Filed (stub) https://github.com/PolymerLabs/arcs/issues/4884 and added todo